### PR TITLE
Avoid need for backstop sync script

### PR DESCRIPTION
**Pull request by davidrapson**
Backstop supports a .js config, so rather than needing to sync .json config to handle different baseUrls I've created a shared config which each variant can call with the appropriate baseUrl.

As a side-effect this also simplifies the pa11y sync script.

### CODE REVIEW

File: testing/wcag/.pa11yci.local.json [hunk 1]
```diff
@@ -10,8 +10,6 @@
     "http://localhost:6006/iframe.html?id=2-design-foundations--colour-language",
     "http://localhost:6006/iframe.html?id=2-design-foundations--colour-palette",
     "http://localhost:6006/iframe.html?id=3-components--advice-feedback",
-    "http://localhost:6006/iframe.html?id=3-components--advice-feedback",
```
Comment (davidrapson) [inline testing/wcag/.pa11yci.local.json:13]: Backstop often uses the same URL for multiple scenarios. I updated the pa11y script to convert the urls to a `Set` first to de-dupe this list. Avoids testing the same URL multiple times.

File: scripts/vr-test.js [hunk 2]
```diff
@@ -7,7 +7,9 @@ const ci = process.argv.some((arg) => arg === '--ci');
 
 const baseCommand =
   'cd ../testing/visual-regression && npx backstop test --docker';
-const command = ci ? `${baseCommand} --config=backstop-ci.json` : baseCommand;
+const command = ci
+  ? `${baseCommand} --config=backstop-config-ci.json`
```
Comment (baseonmars) [inline scripts/vr-test.js:11]: Whis should be `backstop-config-ci.js` right?

Comment (davidrapson) [reply scripts/vr-test.js]: Yes it should!

File: scripts/update-pa11y-config.js [hunk 3]
```diff
@@ -11,72 +11,54 @@
  * the visual regression tests. (.pa11yci.ci.json)
  *
  * Options for both configs can be added into the baseConfig object.
+ * @ TODO: Upgrade storybook to 6 and use JSON export
```
Comment (luke-hill) [inline scripts/update-pa11y-config.js:14]: Storybook is v6 isnt it?

**Review by luke-hill — APPROVED** [review]

File: scripts/update-pa11y-config.js (hunk 3 above)
Comment (davidrapson) [reply scripts/update-pa11y-config.js:14]: This TODO was already here, I just moved it. We are on v6 so whilst we could probably do this now it was beyond the scope of this PR.

**Pull request merged**